packages/functional-tests/tests/react-conversion/oauthSignup.spec.ts

```diff
@@ -101,16 +101,22 @@ test.describe('severity-1 #smoke', () => {
       test.fixme(true, 'Fix required as of 2024/06/28 (see FXA-10003).');
       const { email, password } =
         testAccountTracker.generateSignupAccountDetails();
-      const customEventDetail = createCustomEventDetail(
-        FirefoxCommand.FxAStatus,
-        {
-          capabilities: {
-            choose_what_to_sync: true,
-            engines: ['bookmarks', 'history'],
+      const customEventDetail: FxAStatusResponse = {
+        id: 'account_updates',
+        message: {
+          command: FirefoxCommand.FxAStatus,
+          data: {
```

Comment:
Should we have a `createWhateverYouWantToNameItData` function to create this object for us and overwrite what we need to? I notice there's a lot of repeats. I guess I like the utility `createCustomEventDetail` offered, but if you think it's better to always be this explicit then I'm fine with that.

Reply:
I felt that in this case being explicit was better.  We could create a function to do this. And I even tried reusing the createCustomEventDetail, but it ended it just felt like I was always overriding certain states and each test became less clear when looking at it, especially months later after you haven't looked at the code in while.